docs/source/cpp/compute.rst

```diff
@@ -1231,6 +1231,28 @@ Containment tests
 * \(8) Output is true iff :member:`MatchSubstringOptions::pattern`
   matches the corresponding input element at any position.
 
+
+Hash Functions
+~~~~~~~~~~~~~~
+
+Not to be confused with the "group by" functions, Hash functions produce an array of hash
+values corresponding to the length of the input. Currently, these functions take a single
+array as input.
+
++---------------+-------+-------------+-------------+---------------+-------+
+| Function name | Arity | Input types | Output type | Options class | Notes |
++===============+=======+=============+=============+===============+=======+
+| hash32        | Unary | Any         | UInt32      |               | \(1)  |
++---------------+-------+-------------+-------------+---------------+-------+
+| hash64        | Unary | Any         | UInt64      |               | \(1)  |
++---------------+-------+-------------+-------------+---------------+-------+
+
+* \(1) The hashing algorithm is "xxHash-like", making some minor trade-offs in favor of
```

Comment:
This must be updated.

Reply:
Updated to exclude the unsupported types and also mention no guarantees about hash stability.